### PR TITLE
feat(interactive): Support Identity step in GIE

### PR DESCRIPTION
**Pull request by Louyk14**
Committed-by: Yunkai Lou from Dev container

## What do these changes do?

support identity step in GIE
add tests related to the identity step

## Related issue number

this update is related to issue #3111

Fixes

### CODE REVIEW

File: interactive_engine/compiler/src/test/java/com/alibaba/graphscope/gremlin/IdentityStepTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class IdentityStepTest {
+    private Graph graph = TinkerFactory.createModern();
+    private GraphTraversalSource g = graph.traversal();
+
+    @Test
+    public void g_V() {
```
Comment (longbinlai) [inline interactive_engine/compiler/src/test/java/com/alibaba/graphscope/gremlin/IdentityStepTest.java:37]: Can you include a test case of something like "g.V().as("a").identity()"?

**Review by longbinlai — COMMENTED** [review]
Must include serveal e2e test cases, including at least the cases:
```
g.V().identity() and
g.V().union(identity(), out())
g.V().out().as('a').identity()
```
Verify their actual outputs on the modern graph.

**Review by longbinlai — DISMISSED** [review]
LGTM

File: interactive_engine/compiler/src/main/java/com/alibaba/graphscope/gremlin/integration/suite/standard/IrGremlinQueryTest.java [hunk 2]
```diff
@@ -311,6 +317,61 @@ public void g_V_group_by_by_mean() {
...
+        Traversal<Vertex, Object> traversal =
+                this.get_g_V_identity_values();
+        this.printTraversalForm(traversal);
+        int counter = 0;
+
+        List<String> expected = Arrays.asList("1", "2", "3", "4", "5", "6");
+
+        while (traversal.hasNext()) {
+            Object result = traversal.next();
+            Assert.assertTrue(expected.contains(result.toString()));
```
[the remark this answers is not in the document]
Comment (shirly121) [reply interactive_engine/compiler/src/main/java/com/alibaba/graphscope/gremlin/integration/suite/standard/IrGremlinQueryTest.java:331]: > Why not collect all results and than compare the equivalence of the expected set and computed set? In addition, is it a good idea to `println()` in a unit test? What `counter` is used for?

`printTraversalForm` is a function provided by the official Gremlin testing suite. It solely outputs some log information, such as test name, etc

**Review by shirly121 — APPROVED** [review]

**Pull request merged**